index.html

```diff
@@ -0,0 +1,164 @@
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <main class="main1">
+        <div class="row introduction">
+          <p>"Welcome to Healthline Nutrition!"</p>
+          <h1 class="secondHeader">
+            The Definitive Guide<br />
+            to Healthy Eating in Real Life
+          </h1>
+
+          <p id="second-p">
+            Healthy eating doesn’t have to be complicated. It’s entirely
+            possible to nourish your body while enjoying the foods you love.
+          </p>
+          <h3>2022.10.15(SAT) ~ 16(SUN)</h3>
+          <h5>@ Do not focus on how much you eat ,Focus on what you eat .</h5>
+        </div>
+        <div class="row mainProgram" id="program">
+          <h3>Main Program</h3>
+          <hr />
+          <div class="col-sm card">
+            <img src="./images/fruits.jpg" alt=" fruits" />
+            <h4 class="card-title">Fruits</h4>
+            <p class="card-text">
+              Fresh, frozen, or canned fruits are great choices. Try fruits
+              beyond apples and bananas such as mango, pineapple or kiwi fruit
+            </p>
+          </div>
+          <div class="col-sm card">
+            <img src="./images/vegtables.jpg" alt=" vegtables" />
+            <h4 class="card-title">Vegtables</h4>
+            <p class="card-text">
+              Fresh, frozen, or canned fruits are great choices. Try fruits
+              beyond apples and bananas such as mango, pineapple or kiwi fruit
+            </p>
+          </div>
+          <div class="col-sm card">
+            <img
+              src="./images/glass-clean-drinking-water-44066082.jpg"
+              alt=" Water"
+            />
+            <h4 class="card-title">Water</h4>
+            <p class="card-text">
+              Consuming 2 liters, or 8 glasses, of water a day, but it can also
+              come from dietary sources, such as fruit and vegetables.
+            </p>
+          </div>
+          <div class="col-sm card cal">
+            <img src="./images/calcium.jpg" alt=" calcuim-rich food image" />
+            <h4 class="card-title">calcium-Rich food</h4>
+            <p class="card-text">
+              In addition to fat-free and low-fat milk, consider low-fat and
+              fat-free yogurts without added sugars.
+            </p>
+          </div>
+          <div class="col-sm card cal">
+            <img src="./images/meat.jpg" alt=" meat picture" />
+            <h4 class="card-title">Meat</h4>
+            <p class="card-text">
+              If your favorite recipe calls for frying fish or breaded chicken,
+              try healthier variations by baking or grilling.
+            </p>
+          </div>
+
+          <p class="text-white" id="wholeProgram">See The Whole Program</p>
```

Comment:
- [ ] Please for this `See The Whole Program` make use of a button instead and make sure that it is centred.
### Your design  👇️
![Screenshot from 2022-10-25 18-37-30](https://user-images.githubusercontent.com/89228743/197854745-f8d3e96a-764e-4c87-9328-ed7f43a7104a.png)
### The design in the template 👇️

![Screenshot from 2022-10-25 18-38-31](https://user-images.githubusercontent.com/89228743/197854934-9d72f8c2-f7a8-4e88-9ae1-ee0a1bd753fb.png)

Reply:
Bro, It is like the design but It is so much responsibility that when you continue to decrease the size of the screen this element increases its height to adjust.